pandas/util/testing.py

```diff
@@ -306,12 +306,21 @@ def _skip_if_scipy_0_17():
         pytest.skip("scipy 0.17")
 
 
-def _skip_if_no_lzma():
+def _check_if_lzma():
     try:
         return compat.import_lzma()
     except ImportError:
-        import pytest
-        pytest.skip('need backports.lzma to run')
+        return False
```

Comment:
yes these should be changed to use ``pexpect = pytest.importorskip('pexpect')``
should be simpler